packages/catalog-search/src/tests/PrequerySearchSuggestionItem.test.jsx

```diff
@@ -0,0 +1,36 @@
+import { renderWithRouter } from '@edx/frontend-enterprise-utils';
+import React from 'react';
+import { screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import '@testing-library/jest-dom/extend-expect';
+import PrequerySearchSuggestionItem from '../PrequerySearchSuggestionItem';
+
+describe('<PrequerySearchSuggestionItem />', () => {
+  test('renders course data', () => {
```

Comment:
nit: should probably rename to something like "sends optimizely event on prequery click" or something